iOS/TvShows/TvShows/Common/Network/NetworkConstants/NetworkConstants.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+enum NetworkConstants: String {
+
+    case base = "https://tv-shows.infinum.academy"
+    case login = "/users/sign_in"
+    case register = "/users"
+    case shows = "/shows"
+
+}
```

Comment:
It might be better to just use static properties here rather than cases since you don't actually use the type, you just access the raw value.